Makefile

```diff
@@ -458,6 +458,9 @@ clean: stop-docker
 	rm -f *.out
 	rm -f *.test
 	rm -f imports/imports.go
+	rm -f cmd/platform/cprofile*.out
+	rm -Rf api/data
+	rm -Rf api4/data
```

Comment:
These files are the bane of my existence. I run `rm -Rf api4/data` _very_ frequently and have been meaning to get around to fixing whatever creates them. >:( Unfortunately this won't help me since I leave the docker containers running forever, but I definitely support the change.

Reply:
Yeah, I'd like it if `make clean` didn't reset the Docker containers, since there's already `make clean-docker` for that purpose. I also avoid using `make clean` because I don't want to lose my test data, but I don't want to break anybody else's flow by changing the way the makefile works.

Reply:
I think the unit tests create then when you run them in a specific directory. (like from within vim). I long ago